tests/unit/restapi/lib/actions.py

```diff
@@ -752,7 +752,7 @@ def remove_tag(
     resource_id: int,
     tag_id: int,
 ) -> TestResponse:
-    """Remove tag tag from the resource with the provided unique ID.
+    """Remove tag from the resource with the provided unique ID.
```

Comment:
initially i worried about this change, but there isn't actually an argument named `tag`, so there's nothing to worry about